submissions/dikun/html-css-popup/index.html

```diff
@@ -0,0 +1,146 @@
+                                </a>
+                            </li>
+                            <li class="popup__item">
+                                <a href="#" class="popup__link">
+                                    <img class="popup__img" src="icons/maps.png" alt="">
+                                    <p>Maps</p>
+                                </a>
+                            </li>
+                            <li class="popup__item">
+                                <a href="#" class="popup__link">
+                                    <img class="popup__img" src="icons/files.png" alt="">
+                                    <p>Files</p>
+                                </a>
+                            </li>
+                            <li class="popup__item">
+                                <a href="#" class="popup__link">
+                                    <img class="popup__img" src="icons/contacts.png" alt="">
+                                    <p>Contacts</p>
+                                </a>
+                            </li>
+                            <li class="popup__item">
+                                <a href="#" class="popup__link">
+                                    <img class="popup__img" src="icons/tanslation.png" alt="">
+                                    <p>Translation</p>
+                                </a>
+                            </li>
+                            <li class="popup__item">
+                                <a href="#" class="popup__link">
+                                    <img class="popup__img" src="icons/videos.png" alt="">
+                                    <p>Videos</p>
+                                </a>
+                            </li>
+                            <li class="popup__item">
+                                <a href="#" class="popup__link">
+                                    <img class="popup__img" src="icons/search.png" alt="">
+                                    <p>Search</p>
+                                </a>
+                            </li>
+                            <li class="popup__item">
+                                <a href="#" class="popup__link">
+                                    <img class="popup__img" src="icons/photos.png" alt="">
+                                    <p>Photos</p>
+                                </a>
+                            </li>
+                            <li class="popup__item">
+                                <a href="#" class="popup__link">
+                                    <img class="popup__img" src="icons/apps.png" alt="">
+                                    <p>Apps</p>
+                                </a>
+                            </li>
+                            <li class="popup__item">
+                                <a href="#" class="popup__link">
+                                    <img class="popup__img" src="icons/documents.png" alt="Documents">
```

Comment:
[Read more](https://htmlacademy.ru/blog/articles/alt-text) about how to populate `alt` attribute. Sorry for russian but it's a good article.